src/resources/ha-style.ts

```diff
@@ -23,6 +23,7 @@ documentContainer.innerHTML = `<custom-style>
       --secondary-text-color: #727272;
       --text-primary-color: #ffffff;
       --disabled-text-color: #bdbdbd;
+      --app-toolbar-text-color: var(--text-primary-color)
```

Comment:
Please place the variables under each other in a separate section

Reply:
Will do

Reply:
Updated location of lines and renamed them from `app-toolbar` to `app-header` to be more descriptive